lib/treatments.js

```diff
@@ -16,17 +16,17 @@ function storage (collection, storage, pushover) {
     obj.created_at = created_at.toISOString();
 
     var preBolusCarbs = '';
-    if (obj.preBolus > 0 && obj.carbs) {
+    if (obj.preBolus != 0 && obj.carbs) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected '!==' and instead saw '!='.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/801462991/issues/source?bid=2452374&fileBranchId=2726176#l19)